release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -3415,11 +3415,13 @@ $ oc adm release info 4.15.18 --pullspecs
 The following enhancements are included in this z-stream release:
 
 [id="ocp-4-15-18-unservable-future-versions-status-condition"]
-===== Introducing the UnservableFutureVersions status condition for SHA1 routes
+===== Blocking upgrades for SHA1 serving certificates on routes and Ingress controllers
 
-* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to {product-version} causes routes with SHA1 certificates to be rejected.
+* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to 4.16 cause routes and Ingress controllers with SHA1 certificates to be rejected.
 +
-This update introduces a new `UnservableInFutureVersions` status condition to routes that contain a SHA1 certificate. Additionally, it adds an `admin-gate` to block upgrades if this new status is present on any route. As a result, if cluster administrators have routes that use SHA1 certificates in {product-title} 4.15, they must either upgrade these certificates to a support algorithm or provide an `admin-ack` for the created `admin-gate`. This `admin-ack` allows administrators to proceed with the upgrade without resolving the SHA1 certificate issues, even though the routes will be rejected. (link:https://issues.redhat.com/browse/OCPBUGS-28928[*OCPBUGS-28928*]).
+Only serving certificates are affected. For routes, this is the certificate specified in `spec.tls.certificate`. For Ingress controllers, this is the serving certificate in the secret specified in `spec.defaultCertificate`. CA certificates using SHA1 are not impacted. However, due to link:https://issues.redhat.com/browse/OCPBUGS-42480[*OCPBUGS-42480*], some Ingress controllers were incorrectly blocking upgrades if the serving certificate was not using SHA1, but the CA certificate was. To resolve this, upgrade to a 4.15 version with this bug fix.
++
+Additionally, this update introduces a new `UnservableInFutureVersions` status condition to routes that contain a SHA1 certificate. It also adds an `admin-gate` to block upgrades if this new status is present on any route. As a result, if cluster administrators have routes that use SHA1 certificates in {product-title} 4.15, they must either upgrade these certificates to a support algorithm or provide an `admin-ack` for the created `admin-gate`. This `admin-ack` allows administrators to proceed with the upgrade without resolving the SHA1 certificate issues, even though the routes will be rejected. (link:https://issues.redhat.com/browse/OCPBUGS-28928[*OCPBUGS-28928*]).
```

Comment:
Typo: "suggest" → "suggested".
```suggestion
Additionally, this update introduces a new `UnservableInFutureVersions` status condition to routes that contain a SHA1 certificate. It also adds an `admin-gate` to block upgrades if this new status is present on any route. As a result, if cluster administrators have routes that use SHA1 certificates in {product-title} 4.15, they must either upgrade these certificates to a supported algorithm or provide an `admin-ack` for the created `admin-gate`. This `admin-ack` allows administrators to proceed with the upgrade without resolving the SHA1 certificate issues, even though the routes will be rejected. (link:https://issues.redhat.com/browse/OCPBUGS-28928[*OCPBUGS-28928*]).
```

Reply:
@gcs278, where do we document the command to ack the gate?  Should be `oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.15-route-config-not-supported-in-4.16":"true"}}' --type=merge`, right?

Reply:
I don't see any product documentation. We just have the enhancement https://github.com/openshift/enhancements/blob/master/enhancements/update/upgrades-blocking-on-ack.md

It actually looks like this admin gate implementation was a bugzilla, and I see one comment about product documentation from Ben: https://bugzilla.redhat.com/show_bug.cgi?id=1978376#c11, but I don't see where it was done.

Do you think it's worth asking the updates team for product documentation?